config/quality/pmd/pmd-ruleset.xml

```diff
@@ -34,6 +35,8 @@
     <rule ref="rulesets/java/logging-java.xml">
         <!-- This rule wasn't working properly and given errors in every var call info -->
         <exclude name="GuardLogStatementJavaUtil" />
+        <!-- May be applicable once there is a propper logger -->
+        <exclude name="AvoidPrintStackTrace" />
```

Comment:
Not sure if I still stand by that opinion 😅 there have been a few PR's since I said that that I accepted which contained a change like this...